cgal.sh

```diff
@@ -66,6 +66,14 @@ cmake . \
       -DCGAL_IGNORE_PRECONFIGURED_MPFR:BOOL=YES \
       ${BOOST_ROOT:+-DBoost_NO_SYSTEM_PATHS:BOOL=TRUE -DBOOST_ROOT:PATH="$BOOST_ROOT"}
 
+sed -ibackup "s/register int/int/g" src/CGAL/Turkowski_numeric_solver.cpp
```

Comment:
Use: `sed -ideleteme -e 's/one/two/g' filename` and then `rm -f *.deleteme`. It's the only way to make it work reliably both with GNU sed and BSD's (e.g. on macOS).